middleware/cors.go

```diff
@@ -0,0 +1,75 @@
+package middleware
+
+import (
+	"log"
+	"net/http"
+	"regexp"
+
+	"github.com/aws/aws-lambda-go/events"
+)
+
+var AllowedOrigins = []*regexp.Regexp{
```

Comment:
- do we want to export this?

Reply:
No, I don't think so we need the pattern somewhere else